weave-js/src/components/PagePanelComponents/Home/HomeFeaturedTemplates.tsx

```diff
@@ -64,7 +64,6 @@ export const HomeFeaturedTemplateDrawer: React.FC<{
           onValueChange={(val: string) => setTabValue(val)}>
           <Tabs.List className="px-16">
             <Tabs.Trigger value="Instructions">1. Instructions</Tabs.Trigger>
-            <Tabs.Trigger value="Select Data">2. Select Data</Tabs.Trigger>
```

Comment:
nit: can we make this a comment-out with a comment saying that once we have the interactive data flow we can re-enable this tab.

Reply:
Resolved!